hack/update-pkg-bundle.sh

```diff
@@ -53,18 +53,21 @@ ensure_unique_deployment() {
   fi
 }
 
-generate_package_manifest() {
+generate_package_bundle() {
   # Generate CSV in config/manifests and add boilerplate back (lost when regenerating)
   operator-sdk generate kustomize manifests -q --apis-dir apis
   hack/boilerplaterize hack/boilerplate/boilerplate.yaml.txt config/manifests/**/*.yaml
 
-  # TODO: packagemanifests is deprecated and scheduled to be remove in 2.0.0
-  kustomize build config/manifests | operator-sdk generate packagemanifests -q \
+  # Generate the new package bundle
+  kustomize build config/manifests | operator-sdk generate bundle -q \
```

Comment:
One thing I want to explicitly call out is that one of the annotations changes from cockroachdb-certified to cockroach-operator, as you can see here: https://github.com/cockroachdb/cockroach-operator/pull/794/files#r733931636

cockroachdb-certified doesn't appear to be anywhere in our codebase but has been automatically propagated from the previous version with every new release. Will this cause any issues?